pkg/patterns/declarative/pkg/watch/dynamic.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package watch
 
 import (
+	"atomic"
```

Comment:
Oddly, for me this is not found, it's in sync/atomic.  I'm not quite sure how anything is working though, if we're importing a non-existent package...

Reply:
Ah, phew ... this is in fact why the test is failing (`pkg/patterns/declarative/pkg/watch/dynamic.go:20:2: package atomic is not in std (/usr/local/go/src/atomic)`).  

The line that the prow viewer highlights is a red-herring (`E0606 16:35:17.369998    7074 memcache.go:287] couldn't get resource list for /v1: the server could not find the requested resource`), though I am curious if we can just support it now!

Reply:
facepalm. thanks for debugging the error and catching this.